### PR TITLE
fix broken links

### PR DESCRIPTION
**Pull request by joel-hamill**
## What are your changes?
fix broken links
## What type of changes does your doc introduce?
- [x] Bug fix
- [ ] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [x] High
- [ ] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [x] I have built locally and tested for broken links and formatting.
- [x] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [ ] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.

### CODE REVIEW

File: 1.9/overview/concepts.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ A DC/OS node is a virtual or physical machine on which a Mesos agent and/or Meso
...
 
 - Each master node contains multiple DC/OS components, including most notably a [Mesos master](#mesos-master) process.
-- Master nodes work in a [quorum](https://en.wikipedia.org/wiki/Quorum_(distributed_computing)) to provide consistency of cluster coordination. To avoid [split brain](https://en.wikipedia.org/wiki/Split-brain_(computing)) cluster partitioning, clusters should always have an odd number of master nodes. For example, having three master nodes allows one to be down; having five master nodes allows two to be down, allowing for failure during a rolling update. Additional master nodes can be added for additional risk tolerance.
+- Master nodes work in a [quorum](https://en.wikipedia.org/wiki/Quorum_(distributed_computing&#41;) to provide consistency of cluster coordination. To avoid [split brain](https://en.wikipedia.org/wiki/Split-brain_(computing&#41;) cluster partitioning, clusters should always have an odd number of master nodes. For example, having three master nodes allows one to be down; having five master nodes allows two to be down, allowing for failure during a rolling update. Additional master nodes can be added for additional risk tolerance.
```
Comment (emanic) [inline 1.9/overview/concepts.md]: I think you need an additional closing paranthesis... ) at the end of these links.

Comment (joel-hamill) [inline 1.9/overview/concepts.md]: @emanic i tested this locally and it worked okay. the `&#41;` is an escape character, used to render the parenthesis in the output: http://www.theukwebdesigncompany.com/articles/entity-escape-characters.php. see this slack discussion https://mesosphere.slack.com/archives/documentation/p1481134441000072.

**Comment by emanic** [comment]
OK looked like an error

**Comment by joel-hamill** [comment]
@emanic thanks for thoroughly reviewing.

**Pull request merged**